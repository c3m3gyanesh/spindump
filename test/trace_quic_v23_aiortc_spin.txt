A test with the Lsquic client against the Aiortc implementation on the web. Using  the command:

   ./lsquic/http_client -H quic.aiortc.org -s quic.aiortc.org:443 -p /50000 
   
which tests a regular connection, and successfully generates spin.
