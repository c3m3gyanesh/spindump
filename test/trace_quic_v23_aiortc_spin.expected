QUIC 172.16.114.251 <-> 34.247.69.99 08071b118b5dd5e5-adc3f0fde564e502337319c86a8f663e6d694f (52120:443) at 1571163023098549 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.247.69.99 08071b118b5dd5e5-08dfc448cce41e4e (52120:443) at 1571163023266039 measurement up right 12881 packets 2 0 bytes 2456 0 bandwidth 2456 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023441247 spinflip up 1-0 initiator packets 8 6 bytes 3299 3923 bandwidth 3299 3923 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023611726 spinflip up 1-0 responder packets 12 20 bytes 3731 20174 bandwidth 3731 20174 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023611726 measurement up right 170479 packets 12 20 bytes 3731 20174 bandwidth 3731 20174 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023613509 spinflip up 0-1 initiator packets 12 27 bytes 3731 28630 bandwidth 3731 28630 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023613509 measurement up left 1783 packets 12 27 bytes 3731 28630 bandwidth 3731 28630 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023613509 measurement up full (initiator) 172262 packets 12 27 bytes 3731 28630 bandwidth 3731 28630 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023774384 spinflip up 0-1 responder packets 18 46 bytes 4091 53010 bandwidth 4091 53010 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023774384 measurement up right 160875 packets 18 46 bytes 4091 53010 bandwidth 4091 53010 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023774384 measurement up full (responder) 162658 packets 18 46 bytes 4091 53010 bandwidth 4091 53010 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023774599 spinflip up 1-0 initiator packets 18 48 bytes 4091 55626 bandwidth 4091 55626 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023774599 measurement up left 215 packets 18 48 bytes 4091 55626 bandwidth 4091 55626 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.247.69.99 071b118b5dd5e51f-08dfc448cce41e4e (52120:443) at 1571163023774599 measurement up full (initiator) 161090 packets 18 48 bytes 4091 55626 bandwidth 4091 55626 note "V23,spinning"
